src/entity/Alerts.ts

```diff
@@ -8,6 +8,12 @@ export class Alerts {
 
   @Column({ type: 'bool', default: true })
   filter: boolean;
+
+  @Column({ type: 'bool', default: true })
+  rank: boolean;
 }
 
-export const ALERTS_DEFAULT: Omit<Alerts, 'userId'> = { filter: true };
+export const ALERTS_DEFAULT: Omit<Alerts, 'userId'> = {
+  filter: true,
+  rank: true,
```

Comment:
Also wondering about the defaults.
In the app we set the default filter to false, should these not also be false?

Reply:
This is the source of truth - for better UX we don't want to turn it on in UI then suddenly false after fetching the actual values (it would flick), and I believe we have decided this to be `true` as default based on your suggestion back then 😅